### PR TITLE
chore(META): add default pnpm flags to npmrc

### PR DESCRIPTION
**Pull request by jvanbruegge**
- [ ] There exists an issue discussing the need for this PR
- [ ] I added new tests for the issue I fixed or built
- [x] I used `pnpm run commit` instead of `git commit`

### CODE REVIEW

**Comment by jvanbruegge** [comment]
@staltz CI wont work unless the big PR is merged as that fixes the CI

File: .travis.yml [hunk 1]
```diff
@@ -4,11 +4,11 @@ node_js:
   - '6.10.1'
 script:
   - export IP_ADDR=$(ip addr | grep eth -A 4 | grep 'inet ' | awk '{ print $2 }' | sed 's/\/..//')
-  - pnpm run build && pnpm run lint && pnpm recursive exec --workspace-concurrency 1 --filter @cycle/* -- ../.scripts/retry.sh pnpm run test-ci
+  - pnpm run build && pnpm run lint && pnpm recursive run test-ci
 before_install:
-  - curl -L https://unpkg.com/@pnpm/self-installer | node
+  - npm i -g pnpm@2.21.0
```
Comment (zkochan) [inline .travis.yml]: you can still use the self-installer if you want

```
curl -L https://unpkg.com/@pnpm/self-installer | PNPM_VERSION=2.21.0 node
```

File: pnpmfile.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+//This is needed so pnpm links the needed webpack loaders in node_modules
```
Comment (staltz) [inline pnpmfile.js]: What webpack loaders? Do we use webpack?

Comment (jvanbruegge) [reply pnpmfile.js]: Copy & paste error

**Comment by staltz** [comment]
:heavy_check_mark:

**Pull request merged**